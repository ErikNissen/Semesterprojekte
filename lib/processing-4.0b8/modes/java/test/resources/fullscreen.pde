version https://git-lfs.github.com/spec/v1
oid sha256:e2d6f2ca35d825c6904cd8edd305f66d78b57f8c2e31bc43b81d3c6f51e73f6a
size 19
