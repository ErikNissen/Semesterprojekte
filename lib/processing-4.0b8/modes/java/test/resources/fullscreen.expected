version https://git-lfs.github.com/spec/v1
oid sha256:e55bf5297cddfb982c004484d3c1e544f15b6eea6698ed6fb840ed05d36af96a
size 791
